CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -65,9 +68,11 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
         if !currentCollectionViewSize.equalTo(lastCollectionViewSize) || lastScrollDirection != scrollDirection || lastItemSize != itemSize {
             switch scrollDirection {
             case .horizontal:
-                let inset = (currentCollectionViewSize.width - itemSize.width) / 2
-                collectionView.contentInset = UIEdgeInsets(top: 0, left: inset, bottom: 0, right: inset)
-                collectionView.contentOffset = CGPoint(x: -inset, y: 0)
+                if isFirstCellCentered {
+                    let inset = (currentCollectionViewSize.width - itemSize.width) / 2
+                    collectionView.contentInset = UIEdgeInsets(top: 0, left: inset, bottom: 0, right: inset)
+                    collectionView.contentOffset = CGPoint(x: -inset, y: 0)
```

Comment:
What does this look like when `isFirstCellCentered` is `false`? Could you provide a gif?